hudi-common/src/main/java/org/apache/hudi/common/table/view/AbstractTableFileSystemView.java

```diff
@@ -453,7 +469,13 @@ protected void ensurePartitionLoadedCorrectly(String partition) {
       if (!isPartitionAvailableInStore(partitionPathStr)) {
         // Not loaded yet
         try {
-          LOG.info("Building file system view for partition ({})", partitionPathStr);
+          // For metadata table, always log at DEBUG. For data table, log at INFO once every 10 times
+          if (metaClient.isMetadataTable()) {
+            LOG.debug("Building file system view for partition ({})", partitionPathStr);
+          } else {
+            partitionViewBuildSamplingLogger.logInfoOrDebug(
+                "Building file system view for partition ({})", partitionPathStr);
+          }
```

Comment:
What's the point of doing log sampling here?  In this case, log entries for a subset of partitions will be missing, which can be more confusing, leading to users thinking that some partitions are not scanned.  Would it be better to simply make this DEBUG level?